api/v1/utils/crypto.js

```diff
@@ -26,5 +26,14 @@ module.exports.hashWeak = function (text) {
  * @return {String} a significantly-long, pseudo-random string
  */
 module.exports.generatePassword = function () {
-	return Math.random().toString(PASSWORD_RADIX).substring(0, PASSWORD_LENGTH);
+	return createRandomString(PASSWORD_LENGTH);
 };
+
+// TODO: Determine the length of the token
+module.exports.generateResetToken = function (){
+	return createRandomString(PASSWORD_LENGTH);
```

Comment:
This has nothing to do with the length of a password. Can we think of a better constant name?